dev/com.ibm.ws.security.wim.core/resources/com/ibm/ws/security/wim/util/resources/WimUtilMessages.nlsprops

```diff
@@ -270,4 +270,36 @@ CANNOT_WRITE_TO_READ_ONLY_REPOSITORY.useraction=Ensure that a write operation is
 
 REQUIRED_IDENTIFIERS_MISSING=CWIML3001W: The required identifiers for the entity are missing.
 REQUIRED_IDENTIFIERS_MISSING.explanation=The identifiers uniqueName and uniqueId are not specified. At least one identifier must be specified.
-REQUIRED_IDENTIFIERS_MISSING.useraction=Ensure that the identifier property is specified for each entity passed in. If an identifier is not specified, add the property and specify either the uniqueName or the uniqueId.
\ No newline at end of file
+REQUIRED_IDENTIFIERS_MISSING.useraction=Ensure that the identifier property is specified for each entity passed in. If an identifier is not specified, add the property and specify either the uniqueName or the uniqueId.
+
+KRB5_LOGIN_FAILED_CACHE=CWIML4553E: Kerberos login failed using Kerberos principal {0} and Kerberos credential cache (ccache) {1}.
+KRB5_LOGIN_FAILED_CACHE.explanation=Either the specified Kerberos principal is invalid, or the Kerberos credential cache (ccache) is invalid or expired.
+KRB5_LOGIN_FAILED_CACHE.useraction=Ensure a valid Kerberos principal is specified and the Kerberos credential cache (ccache) has not expired.
+
+KRB5_LOGIN_FAILED_KEYTAB=CWIML4554E: Kerberos login failed using Kerberos principal {0} and Kerberos keytab {1}.
+KRB5_LOGIN_FAILED_KEYTAB.explanation=Either the specified Kerberos principal is invalid, or the Kerberos keytab is invalid.
+KRB5_LOGIN_FAILED_KEYTAB.useraction=Ensure a valid Kerberos principal is specified and a valid Kerberos keytab containing the Kerberos principal.
+		
+KRB5_LOGIN_FAILED_DEFAULT_CACHE=CWIML4555E: Kerberos login failed using Kerberos principal {0} and the default Kerberos credential cache (ccache).
+KRB5_LOGIN_FAILED_DEFAULT_CACHE.explanation=Either the specified Kerberos principal is invalid, or the default Kerberos credential cache (ccache) is invalid or expired.
+KRB5_LOGIN_FAILED_DEFAULT_CACHE.useraction=Ensure a valid Kerberos principal is specified and the default Kerberos credential cache (ccache) has not expired.
+
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB=CWIML4556E: Kerberos login failed using Kerberos principal {0} and the default Kerberos keytab.
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB.explanation=Either the specified Kerberos principal is invalid, or the default Kerberos keytab is invalid.
+KRB5_LOGIN_FAILED_DEFAULT_KEYTAB.useraction=Ensure a valid Kerberos principal is specified and the default Kerberos keytab is valid.
```

Comment:
Ensure that a valid Kerberos principal is specified and that the default Kerberos keytab is valid.